cpp/example-control.cc

```diff
@@ -0,0 +1,32 @@
+#include <iostream>
+
+#include "zendar-api.h"
+#include "reqrep.pb.h"
+
+int main(int argc, char* argv[]) {
+  if (argc < 2) {
+    printf("Usage: example-control URI");
+  }
+  zendar::ZendarReceiver rcv(argv[1]);
+  zendar::ZendarError error;
+  zen_proto::control::Response rep;
+  rcv.Connect();
```

Comment:
Does `Connect()` implicitly do a `Start()`?

Reply:
Discussed in our 1-on-1. The Stop is there to reset the ZPU.